clang/test/dpct/helper_files_ref/include/math.hpp

```diff
@@ -149,6 +149,19 @@ compare(const T a, const T b, const BinaryOperation binary_op) {
   return {compare(a[0], b[0], binary_op), compare(a[1], b[1], binary_op)};
 }
 
+/// Performs 2 element comparison and return an unsigned int.
```

Comment:
performs 2 elements comparison and returns ..

Reply:
fixed